Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/introduction/structure.tex

The paper is organized as follows:

> In \autoref{sec:related-works}, articles related to topic modeling and random walk are investigated.

Author: ```suggestion
In \autoref{sec:related-works}, we mention previous articles related to topic modeling, \gls{ir}, and random walk.
```
```suggestion
In \autoref{sec:related-works}, we reference previous articles related to topic modeling, \gls{ir}, and random walk.
```

\autoref{sec:dataset} describes the dataset used in the experiments.

> In \autoref{sec:method}, the steps of our method are shown, and \autoref{sec:prepro}, \autoref{sec:lm}, \autoref{sec:lda}, \autoref{sec:query}, and \autoref{sec:pagerank} describe these steps further.

Author: ```suggestion
\autoref{sec:method} explains the steps of our method, while \autoref{sec:prepro}, \autoref{sec:lm}, \autoref{sec:lda}, \autoref{sec:query}, and \autoref{sec:pagerank} describe these steps in further detail.
```

> In \autoref{sec:experiment}, the test combinations and hyperparameters are detailed and results are shown.

Author: ```suggestion
In \autoref{sec:experiment} we setup an experiment to test the performance of different combinations of \gls{ir} methods.
```

> \autoref{sec:discussion} analyzes and discusses these results.

Author: ```suggestion
In \autoref{sec:discussion} we analyze and discusses the results of our experiment.
```
